lib/baby_squeel/compat.rb

```diff
@@ -163,5 +170,24 @@ def where(*args, &block)
         end
       end
     end
+
+    module Resolver
+      private
+
+      def build(strategy, name, *args)
+        result = super(strategy, name, *args)
+        result.nil? && strategy == :table ? Table.new(Table.new(Arel::Table.new(name))) : result
```

Comment:
Is there a reason to double wrap the `Table`? Should this just be `Table.new(Arel::Table.new(name))`?

Reply:
No, oops...